ScreenRuler/SetSizeForm.resx

```diff
@@ -267,17 +267,17 @@
     <value>True</value>
   </metadata>
   <data name="$this.AutoScaleDimensions" type="System.Drawing.SizeF, System.Drawing">
-    <value>6, 13</value>
+    <value>6, 12</value>
   </data>
   <data name="$this.ClientSize" type="System.Drawing.Size, System.Drawing">
-    <value>214, 131</value>
+    <value>214, 121</value>
   </data>
   <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
   <data name="$this.StartPosition" type="System.Windows.Forms.FormStartPosition, System.Windows.Forms">
     <value>CenterParent</value>
   </data>
   <data name="$this.Text" xml:space="preserve">
-    <value>Set Ruler Length</value>
+    <value>设置标尺长度</value>
```

Comment:
`SetSizeForm.zh.resx`?

Reply:
Hello alxnull，I have added the translation in a separate file *.resx.